src/utils/urls.js

```diff
@@ -0,0 +1,14 @@
+export const getLoginUrl = () => (
+  `${process.env.REACT_APP_EGGHEAD_BASE_URL}/users/jwt?return_to=${window.location.href}`
+)
+
+export const guideUrl = 'https://instructor.egghead.io/guide'
+
+export const chatUrl = 'https://eggheadio.slack.com/messages/egghead-instructors/'
```

Comment:
@trevordmiller Cool, It's just that there are two different paragraphs one right above the other that both lead the user to the same place